.github/workflows/dockerhub.yml

```diff
@@ -0,0 +1,19 @@
+name: Docker Image CI
```

Comment:
Can we give this a better name? Maybe `Build Docker`?
```suggestion
name: Build Docker
```

Reply:
Updated name to Build Docker Image